kalite/distributed/features/steps/generic.py

```diff
@@ -1,6 +1,9 @@
 from behave import *
 from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
 
 @given("I am on the homepage")
 def step_impl(context):
-    go_to_homepage(context)
\ No newline at end of file
+	if not User.objects.exists():
```

Comment:
if we create a new step like this, we need to manually update all the affected tests. Seems anyone working in other feature don't want to be blocked by superuser modal, making superuser always exists so that they don't even need to know to include @given("There is a superuser")